src/components/Avatar/__stories__/avatar.stories.mdx

```diff
@@ -0,0 +1,205 @@
+<Meta 
+  title="Components/Avatar" 
+  component={Avatar} 
+/>
+
+<!--- Component template -->
+
+export const avatarTemplate = createComponentTemplate(Avatar);
+
+<!--- Component documentation -->
+
+# Avatar
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Avatar is a graphical representation of a person through a profile picture, image, icon, or set of initials.
+
+<Canvas>
+  <Story name="Overview" args={{size: Avatar.sizes.Large, src: person1, type: Avatar.types.IMG }}>>
+      {avatarTemplate.bind({})}
+  </Story>
+</Canvas> 
+
+## Props
+<ArgsTable of={Avatar} />
+
+## Usage
+<UsageGuidelines
+  guidelines={[
+    "Use an avatar to help a user in the platform efficiently identify another person or a team.",
+    "When there is no personal photo to show, use initials.",
+    "If an image fails to load, fall back to the default user avatar.",
+    "An avatar may contain a status icon to indicate a user’s status (working from home, busy, etc.).",
+    "Use a tooltip or dialog when hovering over the avatar to offer more information. For example: with a person’s name."
+  ]}
+/>
+
+## Variants
+### Size
+Avatars appear in 3 sizes: Small, Medium, and Large.
+<Canvas>
+  <Story name="Size">
+    <Avatar size={Avatar.sizes.SMALL} src={person1} type={Avatar.types.IMG} />
+    <Avatar size={Avatar.sizes.MEDIUM} src={person1} type={Avatar.types.IMG} />
+    <Avatar size={Avatar.sizes.LARGE} src={person1} type={Avatar.types.IMG} />
+  </Story>
+</Canvas>
+
+### Disabled
+Use when a user is inactive in the system.
+<Canvas>
+  <Story name="Disable">
+    <Avatar size={Avatar.sizes.SMALL} src={person1} type={Avatar.types.IMG} isDisabled />
+    <Avatar size={Avatar.sizes.MEDIUM} src={person1} type={Avatar.types.IMG} isDisabled />
+    <Avatar size={Avatar.sizes.LARGE} src={person1} type={Avatar.types.IMG} isDisabled />
+  </Story>
+</Canvas>
+
+### Avatar with text
+Use when a user’s image is not available, use their initials.
+<Canvas>
+  <Story name="Avatar with text">
+    <Avatar size={Avatar.sizes.SMALL} type={Avatar.types.TEXT} text="RM" backgroundColor={Avatar.colors.LIPSTICK} />
+    <Avatar size={Avatar.sizes.MEDIUM} type={Avatar.types.TEXT} text="RM" backgroundColor={Avatar.colors.LIPSTICK} />
+    <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.TEXT} text="RM" backgroundColor={Avatar.colors.DONE_GREEN} />
+  </Story>
+</Canvas>
+
+### Square avatar with icon or text
+Use for non-person avatars, such as a workspace or team.
+<Canvas>
+  <Story name="Square avatar">
+    <Avatar
+      size={Avatar.sizes.SMALL}
+      type={Avatar.types.TEXT}
+      text="R"
+      backgroundColor={Avatar.colors.BRIGHT_BLUE}
+      isSquare
+    />
+    <Avatar
+      size={Avatar.sizes.MEDIUM}
+      type={Avatar.types.ICON}
+      text="RM"
```

Comment:
please remove the usage on the text prop since there is no need in it in the Icon avatar